tests/ui_tests/test_cluster_list_page.py

```diff
@@ -0,0 +1,85 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import allure
+import pytest
+from adcm_client.objects import (
+    ADCMClient,
+    Bundle,
+)
+from adcm_pytest_plugin import utils
+
+from tests.ui_tests.app.page.cluster_list.page import ClusterListPage
+
+
+@allure.step("Upload cluster bundle")
+def cluster_bundle(sdk_client_fs: ADCMClient, data_dir_name: str) -> Bundle:
+    return sdk_client_fs.upload_from_fs(os.path.join(utils.get_data_dir(__file__), data_dir_name))
+
+
+@pytest.mark.parametrize("bundle_archive",
+                         [
+                             pytest.param(utils.get_data_dir(__file__, "cluster_community"),
+                                          id="community"),
+                             pytest.param(utils.get_data_dir(__file__, "cluster_enterprise"),
+                                          id="enterprise")
+                         ],
+                         indirect=True)
+def test_check_cluster_list_page_with_cluster_creating(app_fs, auth_to_adcm, bundle_archive):
+    edition = bundle_archive.split("cluster_")[2][:-4]
+    cluster_params = {"bundle": f"test_cluster 1.5 {edition}",
+                      "description": "Test", "state": "created"}
+    cluster_page = ClusterListPage(app_fs.driver, app_fs.adcm.url).open()
+    with allure.step("Check no cluster rows"):
+        assert len(cluster_page.table.get_all_rows()) == 0, "There should be no row with clusters"
+    # pylint: disable=simplifiable-if-expression
+    cluster_page.upload_cluster(bundle_archive, cluster_params['description'],
+                                is_license=True if edition == "enterprise" else False)
+    with allure.step("Check uploaded cluster"):
+        assert len(cluster_page.table.get_all_rows()) == 1, "There should be 1 row with cluster"
```

Comment:
We may need CommonTableObj.rows_count() method, because it looks useful.
Can be implemented later though

Reply:
yes, let's implement methods when they are necessary